### ISSUE TITLE
When used in conjunction with Duo 2 factor auth plugin it requires two logins

### ISSUE THREAD

**Issue by Teejer** [NeosIT/active-directory-integration2#69]
We are a Duo Security customer and would like to enable two factor auth on our wordpress. When we enable the duo plugin and the Next AD plugin it requires users to login two times. If we disable next ad then it only requires one login.

Here is the link for the duo plugin. https://wordpress.org/plugins/duo-wordpress/

**Comment by Teejer** [comment]
wordpress version 4.9.2/NADI 2.1.0/Duo Two-Factor Authentication 2.5.4

**Comment by MeissnerDa** [comment]
Hello @Teejer,

I am sorry to hear that NADI has conflicts with the plugin https://wordpress.org/plugins/duo-wordpress/. At the moment we do not have the manpower to take a look at how the two plugins could work together so I can not give you any ETA about a fix / workarround.

Best regards,
meissnerDa

**Comment by sheneman** [comment]
We are experiencing this same issue.   Has this been resolved?

**Comment by schakko** [comment]
@sheneman This has not been solved yet due to other priorities.

**Comment by schakko** [comment]
I've added a short survey at https://docs.google.com/forms/d/1ZV9IXpzhcGNf3VrYh_HaNg6yaEC6t2rdMXM5yMCZbEo to see if there is enough interest in a solution/sponsoring for this issue.

@Teejer @sheneman @photuris5 I'd happy if you let me know your thoughts about it.

**Comment by kathyIT** [comment]
Are there any updates to this issue? We are experiencing the same issue after installing the Duo plugin.

**Comment by schakko** [comment]
Nope, no updates. Without any sponsorship we probably won't dig further into this issue.

**Comment by kathyIT** [comment]
Are you aware of any security caused between your plugin and the Duo plugin if we continue to use Duo?

Thanks,

Kathy Dannenberg
Information Security Manager
Southwestern Illinois College
618-641-5122

[swic security]<https://infoshare.swic.edu/sites/dept/IT/ITDeptSite/Security/SitePages/Home.aspx>

**Comment by schakko** [comment]
I am not aware of any issues but we had no external audit if this is something you require.

**Comment by Teejer** [comment]
If it helps we ended up using the duo ldap proxy to get around this issue.

It's not a perfect solution. It's an "auto push" at successful ldap bind a
duo push is automatically sent to the users device.

**Comment by Teejer** [comment]
And the login page just sits there until the user accepts/allows the duo
push.

**Comment by kathyIT** [comment]
@Teejer 
Thank you for the suggestion. We will check into it.
